tests/integration_tests/conftest.py

```diff
@@ -176,6 +199,52 @@ async def token(request):
     await runner.cleanup()
 
 
+@pytest.fixture
+async def jwks_server():
+    host = "localhost"
+    port = 4080
+
+    app = web.Application()
+    routes = web.RouteTableDef()
+
+    class Handle(object):
+        def __init__(self):
+            self.host = host
+            self.port = port
+            self.result = {
+                "keys": [{
+                    "kty": "RSA",
+                    "e": "AQAB",
+                    "use": "sig",
+                    "kid": "L7wdUtrDssMTb57A_TNAI79DQCdp0T2-KUrSUoDJBhk",
```

Comment:
Use the fixture here too. You can merge dictionaries like this:
```python3
>>> header = {"kid": "whatever"}
>>> {"foo": "bar", **header}
{'foo': 'bar', 'kid': 'whatever'}
```
But maybe it makes more sense to have the fixture just return the kid and format it appropriately wherever it's used.

Reply:
Makes sense